src/main/java/com/celements/web/service/IWebUtilsService.java

```diff
@@ -138,6 +143,12 @@ public String getDocSection(String regex, DocumentReference docRef, int section)
   public EntityReference resolveEntityReference(String name, EntityType type,
       WikiReference wikiRef);
 
+  public <T extends EntityReference> T resolveReference(@NotNull String name,
```

Comment:
please add JavaDoc explaining shortly the usage and hint to resolveSpaceReference, resolveDocumentReference, resolveAttachmentReference and resolveWikiReference